### ISSUE TITLE
saving microphone audio

### ISSUE THREAD

**Issue by eli99999** [Azure-Samples/Cognitive-Speech-STT-Android#42]
how can i use the microphone and save the audio stream to a wav file or a video file as well?

**Comment by priyaravi20** [comment]
This is not something supported by API. But you should be able to use the native android audio drivers to do audio handling operations.

**Comment by eli99999** [comment]
OK, let me rephrase my question:
is there a way to get the audio stream as an output form the MicrophoneRecognitionClient class?
in any way I'm trying to sent the audio bytes that i get from other andtoid classes(AudioRecord) as PCM stream with the DataRecognitionClient's sendAudio fuction and i'm getting an error from the sdk(stream in worng format)
does the class need's to have the pcm data with wav headers or can it handle raw pcm data?

**Comment by priyaravi20** [comment]
It requires the riff wav headers.
